lib/modules/datasource/conda/index.spec.ts

```diff
@@ -61,5 +62,46 @@ describe('modules/datasource/conda/index', () => {
       expect(res).toMatchSnapshot();
       expect(res?.releases).toHaveLength(94);
     });
+
+    it('returns null without registryUrl', async () => {
+      const condaDatasource = new CondaDatasource();
+      const res = await condaDatasource.getReleases({
+        registryUrl: '',
+        packageName: depName,
+      });
+      expect(res).toBeNull();
+    });
+
+    it('supports multiple custom datasource urls', async () => {
+      const depName = 'pytest';
+      httpMock
+        .scope('https://api.anaconda.org/package/rapids')
+        .get(`/${depName}`)
+        .reply(404);
+      httpMock
+        .scope('https://api.anaconda.org/package/conda-forge')
+        .get(`/${depName}`)
+        .reply(200, {
+          html_url: 'http://anaconda.org/anaconda/pytest',
+          dev_url: 'https://github.com/pytest-dev/pytest/',
+          versions: ['2.7.0', '2.5.1', '2.6.0'],
+        });
+      const config = {
+        registryUrls: [
+          'https://api.anaconda.org/package/rapids',
+          'https://api.anaconda.org/package/conda-forge',
+          'https://api.anaconda.org/package/nvidia',
+        ],
+      };
+      const res = await getPkgReleases({
+        ...config,
+        datasource,
+        depName,
+      });
+      expect(res?.registryUrl).toBe(
+        'https://api.anaconda.org/package/conda-forge'
+      );
+      expect(res?.releases).toHaveLength(3);
```

Comment:
please compare full object in single expectation, should be pretty small.

Reply:
fixed !